app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError("All friends should be vaccinated")

```

Comment:
The error message here could be more descriptive. Instead of 'All friends should be vaccinated', consider something like 'The visitor is not vaccinated. All visitors must be vaccinated to enter the cafe.' This gives a clearer explanation of what the problem is.